kubejs/server_scripts/mods/modern_industrialization/blast_furnace.js

```diff
@@ -39,6 +39,17 @@ ServerEvents.recipes(e => {
         e.custom(newRecipe).id(id);
     }
 
+    // -- IMPURRE LIQUID NETHER STAR -> MOLTEN NETHER STAR -- //
+    blastFurnace(
```

Comment:
Potential to add a catalyst for this recipe to aid in the "purifying" of the nether star. Unsure what would be best off the top of my head.

Reply:
I like platinum dust for that. Set consumption probability to 0.05, and then it "has to be replaced every once in a while".

Reply:
I'm not a fan of probability inputs in the blast furnace. How about it just be 1 tiny platinum dust?